sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,20 +1,98 @@
 # Release History
 
-## 2.0.0-beta.7 (Unreleased)
+## 2.0.0 (2021-10-12)
+
+We're proud to announce the first GA release of the `@azure/identity` v2 package. The v2 package includes the best parts of v1, plus several improvements.
+
+This changelog entry showcases the changes that have been made. For a v1-to-v2 migration guide, see the [migration guide](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/migration-v1-v2.md).
 
 ### Features Added
 
+#### Plugin API
+
+Identity v2 provides a top-level `useIdentityPlugin` method, which allows using two new plugin packages:
+
+- [@azure/identity-vscode](https://www.npmjs.com/package/@azure/identity-vscode), which provides the dependencies of `VisualStudioCodeCredential` and enables it.
+- [@azure/identity-cache-persistence](https://www.npmjs.com/package/@azure/identity-cache-persistence), which provides persistent token caching.
+- If the `@azure/identity-vscode` plugin isn't used through the `useIdentityPlugin` function, the `VisualStudioCodeCredential` exposed by Identity v2 will throw a `CredentialUnavailableError`.
+
+Most credentials on Identity v2 now include a constructor option called `tokenCachePersistenceOptions`. If set to `tokenCachePersistenceOptions: { enabled: true }`, it will enable persistent token caching on that credential instance.
+
+#### New credentials
+
+Identity v2 includes three new credential types:
+
+- `AzurePowerShellCredential`, which re-uses any account previously authenticated with the `Az.Account` PowerShell module.
+- `ApplicationCredential`, which is a simplified `DefaultAzureCredential` that only includes `EnvironmentCredential` and `ManagedIdentityCredential`.
+- `OnBehalfOfCredential`, which enables the [On-Behalf-Of authentication flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow).
+
+#### Changes to all credentials
+
+Identity v2 enables:
+
+- Support for claims challenges resulting from [Continuous Access Enforcement (CAE)](https://docs.microsoft.com/azure/active-directory/conditional-access/concept-continuous-access-evaluation) and [Conditional Access authentication context](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/granular-conditional-access-for-sensitive-data-and-actions/ba-p/1751775). To disable this behavior, set the environment variable `AZURE_IDENTITY_DISABLE_CP1` to any value. For more about claims challenges, see [Claims challenges, claims requests, and client capabilities](https://docs.microsoft.com/azure/active-directory/develop/claims-challenge).
```

Comment:
I thought CAE was something that the consuming libraries implements and the Identity package was oblivious to it

Reply:
It is both!

Reply:
So, this is relevant for package owners, but not necessarily to end users? Regarding the note on disabling this feature, can we add something that sheds light on when someone would want to disable this feature?

Reply:
Users working with these credentials directly already have to figure out their own way to refresh tokens. The default change in v2 is that now every token they will get will have been gotten through a process that tells AAD to try to initiate the claims challenge flow when te token expires. This can be disabled through the environment variable in case the users don’t want Azure to answer to requests with expired tokens with the challenge claims.

🤔 I’ll put something like that.

Reply:
Let me know how it looks now!